internal/testplanet/node.go

```diff
@@ -86,7 +86,8 @@ func (node *Node) initOverlay(planet *Planet) error {
 	}
 
 	node.Kademlia = kad
-	node.Overlay = overlay.NewOverlayCache(teststore.New(), node.Kademlia)
+	// TODO(moby) pass sdbclient
```

Comment:
just wondering, why does this have to be passed in the future but not now?

Reply:
This is like one of the last things I have to do before it's actually ready to merge